api/db/populate_users.js

```diff
@@ -0,0 +1,121 @@
+#! /usr/bin/env node
+
+// TODO: This will all be in the general js to abide by DRY principal
+const userArgs = process.argv.slice(2);
+
+if (!userArgs[0].startsWith('mongodb')) {
+  console.log(
+    'Error: you need to specify a valid MongoDB URL as the first argument'
+  );
+
+  return;
+}
+
+const User = require('../models/User');
+const Interest = require('../models/Interest')
```

Comment:
I'll take a look at the settings. I don't recall getting any warnings/errors regarding it. I am using the same development settings that I have for work, which may be causing Prettier not to be working correctly.

Reply:
I just pushed up all the updates I've made so far for populating the users. When you get some free time, do you mind checking if the Prettier linting looks better?

Reply:
Looks much better, although there are still some things which aren't as they should, for instance the `"`s should be `'`s, and there are still missing semicolons 😕

Reply:
Just pushed up the reformatted code. Hopefully I've gotten the linter sorted out as it should.

Reply:
It only gives me two changes now: the semicolons on current lines 12 and 116 shouldn't be there.

By the way, do you know any way I can make it easier for other contributors to not face these problems? I can see how this can be frustrating for people wanting to contribute 😞

Reply:
Not sure exactly. When I run the commit, I get this error: 
`Some of your tasks use `git add` command. Please remove it from the config since all modifications made by tasks will be automatically added to the git commit index.`

Reply:
I'll fix that now, we'll see if it helps! 🙂